plugins/node/opentelemetry-instrumentation-aws-lambda/README.md

```diff
@@ -69,6 +70,18 @@ new AwsLambdaInstrumentation({
 })
 ```
 
+### Specifying the Lambda Handler
+
+The instrumentation will attempt to automatically determine the Lambda handler function to instrument. To do this, it relies on the `_HANDLER` environment variable which is [set by the Lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime). For most use cases, this will accurately represent the handler that should be targeted by this instrumentation.
+
+There exist use cases where the instrumetation's default behavior will not work. For these use cases, the `lambdaHandler` option should be used to explicitly specify the Lambda handler.
```

Comment:
```suggestion
There exist use cases where the `_HANDLER` environment variable does not accurately represent the module that should be targeted by this instrumentation. For these use cases, the `lambdaHandler` option can be used to explicitly specify the Lambda handler that should be instrumented.
```

I actually prefer the wording you used in the PR description to describe this particular use case since it explains the issue more clearly than saying the instrumentation's default behavior doesn't work, which is more vague.